internal/common/util/map.go

```diff
@@ -58,3 +58,11 @@ func FilterKeys(a map[string]string, keys []string) map[string]string {
 
 	return result
 }
+
+func InverseMap[K comparable, V comparable](a map[K]V) map[V]K {
```

Comment:
We probably should put some doc on this function to say what it does in the case of duplicate keys,